### PR TITLE
Update hadoop libraries

### PR DESCRIPTION
**Pull request by arhimondr**
Latest version of `hadoop-apache2` library contains a fix for Hadoop `FileSystem`
cache leak in case of Kerberos authentication with impersonation.

### CODE REVIEW

**Comment by arhimondr** [comment]
@electrum

**Comment by electrum** [comment]
Don't use backticks in commit messages for things like "FileSystem" since commit messages aren't markdown.

**Comment by electrum** [comment]
Typo "Haddop" in commit message

File: presto-product-tests/conf/presto/etc/catalog/hive.properties [hunk 1]
```diff
@@ -13,3 +13,5 @@ hive.allow-rename-column=true
 hive.allow-drop-table=true
 hive.allow-rename-table=true
 hive.metastore-cache-ttl=0s
+
```
Comment (electrum) [inline presto-product-tests/conf/presto/etc/catalog/hive.properties]: No need for blank line here

File: presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-impersonation/hive.properties [hunk 2]
```diff
@@ -22,3 +22,5 @@ hive.hdfs.presto.keytab=/etc/hadoop/conf/hdfs.keytab
 
 #required for testGrantRevoke() product test
 hive.security=sql-standard
+
+hive.fs.cache.max-size=10
```
Comment (electrum) [inline presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-impersonation/hive.properties]: Let's move this above after the other HDFS configs

File: presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-no-impersonation/hive.properties [hunk 3]
```diff
@@ -23,3 +23,6 @@ hive.hdfs.authentication.type=KERBEROS
 hive.hdfs.impersonation.enabled=false
 hive.hdfs.presto.principal=hdfs/hadoop-master@LABS.TERADATA.COM
 hive.hdfs.presto.keytab=/etc/hadoop/conf/hdfs.keytab
+
+hive.fs.cache.max-size=10
+
```
Comment (electrum) [inline presto-product-tests/conf/presto/etc/environment-specific-catalogs/singlenode-kerberos-hdfs-no-impersonation/hive.properties]: Remove trailing blank line

**Comment by electrum** [comment]
A few minor comments, otherwise looks good

**Comment by arhimondr** [comment]
@electrum updated

**Comment by electrum** [comment]
Merged, thanks!

**Pull request closed without merging**